datanode/sqlstore/epoch_test.go

```diff
@@ -61,13 +65,19 @@ func TestEpochs(t *testing.T) {
 	epoch3Start := epoch2End
 	epoch3Expire := epoch3Start.Add(time.Minute)
 
+	block1 := addTestBlockWithVegaTime(t, bs, epoch1Start)
```

Comment:
All of these insert blocks with height 2 right? So for `epoch1b` where we should be testing a case where `FirstBlock` and `LastBlock` are different, we're not because they're both height 2.